3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -264,12 +270,23 @@ object EnrichmentManager {
       }
     }
 
-    // Finally anonymize the IP address
+    // To anonymize the IP address
     Option(event.user_ipaddress).map(ip => event.user_ipaddress = registry.getAnonIpEnrichment match {
       case Some(anon) => anon.anonymizeIp(ip)
       case None => ip
     })
 
+    // Finalize the currency conversion
+    val currency = {
+      registry.getCurrencyConversionEnrichment match {
+        case Some(currency) => {
+          (trTotal |@| trTax |@| trShipping |@| tiPrice) { 
+            currency.convertCurrencies(Option(event.tr_currency), _, _, _, Option(event.ti_currency), _, Option(event.co_tstamp))
```

Comment:
I'm puzzled how this compiles. The POJO field is called event.collector_tstamp and it is a String, not a Joda DateTime. What am I missing?